mlbstatsapi/mlb_api.py

```diff
@@ -124,18 +131,99 @@ def get_person(self, player_id, **params) -> Union[Person, None]:
             for person in mlb_data.data['people']:
                 return Person(**person)
 
+    def get_persons(self, person_ids, **params) -> List[Person]:
+        """
+        This endpoint returns statistical data and biographical information 
+        for a players,umpires, and coaches based on playerId.
+
+        Parameters
+        ----------
+        person_ids : str
+            Insert personId(s) to return biographical information for a 
+            specific player. Format '605151,592450'
+
+        Other Parameters
+        ----------------
+        hydrate : str
+            Insert hydration(s) to return statistical or biographical data 
+            for a specific player(s). 
+            Format stats(group=["statGroup1","statGroup2"],
+                         type=["statType1","statType2"]).
+            Available Hydrations:
+                hydrations
+                # awards
+                currentTeam
+                team
+                rosterEntries
+                relatives
+                transactions
+                social
+                education
+                # stats
+                draft
+                mixedFeed
+                articles
+                videos
+                xrefId
+        fields : str
+            Comma delimited list of specific fields to be returned. 
+            Format: topLevelNode, childNode, attribute
+
+        Returns
+        -------
+        Person
+            Returns a Person
+
+        See Also
+        --------
+        Mlb.get_people : Return a list of People from sport id.
+        Mlb.get_people_id : Return person id from name.
+
+        Examples
+        --------
+        >>> mlb = Mlb()
+        >>> mlb.get_person(660271)
+        Person
+        """
+        if 'stats' in params:
+            return []
+        elif 'awards' in params:
+            return []
+
+        params['personIds'] = person_ids
```

Comment:
Why plural?

Reply:
The get command is get_persons, Insert personId(s) to return biographical information for a specific player. Format '605151,592450'.

Reply:
And we already have a get_person. This get is to grab multiple people at one time, allowing the user to save on making multiple, calls to the api

Reply:
Oh nice so you can pass two people IDs to the people endpoint?